app/models/evaluation_result.py

```diff
@@ -21,9 +22,23 @@ class EvaluationResult(db.Model):
     model_id = Column(Integer, nullable=False)
     data_path = Column(String(512), nullable=False)
     evaluation_id = Column(Integer, ForeignKey('evaluations.evaluation_id'), nullable=False)
-    result = Column(Text, nullable=False)
+    _result = Column(Text, nullable=False)
     register_date = Column(DateTime, default=datetime.datetime.utcnow, nullable=False)
 
+    @property
+    def result(self):
+        res = json.loads(self._result)
+        res['result_id'] = self.evaluation_result_id
+        return res
+
+    @result.setter
+    def result(self, value):
+        if isinstance(value, dict):
+            res = json.dumps(value)
+        else:
+            res = value
+        self._result = res
```

Comment:
This is to

- get `result` as dict in Python while storing in db as string.
- `result` contains `result_id` in the dict